canonical-websites/www.ubuntu.com
# Update the Linkedin link in the footer

## Done
Update the Linkedin link in the footer

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Click the Linkedin icon in the footer and see it goes to: https://www.linkedin.com/company/ubuntu/

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4052
